crates/redis_interface/src/commands.rs

```diff
@@ -371,6 +371,19 @@ impl super::RedisConnectionPool {
         Ok(hsetnx)
     }
 
+    #[instrument(level = "DEBUG", skip(self))]
+    pub async fn increment_field_in_hash(
+        &self,
+        key: &str,
+        field: &str,
+        increment: i64,
+    ) -> CustomResult<usize, errors::RedisError> {
+        self.pool
+            .hincrby(self.add_prefix(key), field, increment)
```

Comment:
do we expect this functions to be scoped under tenants?

Reply:
the aggregates will be stored per merchant basis and a merchant will belong to tenant right so yes it should be scoped under tenants